Makefile

```diff
@@ -18,7 +18,8 @@ TEST_PATHS:=commands repos/common $(SYSUPG_TEST_PATHS)
 
 
 ifdef ACTOR
-	TEST_PATHS=`python utils/actor_path.py $(ACTOR)`
+	TEST_PATHS=`$(_PYTHON_VENV) utils/actor_path.py $(ACTOR)`
+	APPROX_TEST_PATHS=$(shell $(_PYTHON_VENV) utils/find_actors.py -C repos $(ACTOR))  # Dev only
```

Comment:
The `_PYTHON_VENV` var is used before it is defined, definition is on line 37. However after I moved the definition it still tries to run all the tests if PYTHON_VENV isn't set.

Reply:
I've done some further tests and for me on python2.7 and python3.6 it attempts to run all the tests, at the beginning of the output I have:
```
. tut/bin/activate; \
${PYTHON_VENV:-python2.7} -m pytest
===================================== test session starts =====================================
platform linux -- Python 3.6.15, pytest-4.6.11, py-1.11.0, pluggy-0.13.1 -- /home/matej/projects/leapp-repo/tut/bin/python3.6
``` 
whereas on python3.9, which works as expected I have:
```
. tut/bin/activate; \
${PYTHON_VENV:-python2.7} -m pytest  /home/matej/projects/leapp-repo/repos/system_upgrade/common/actors/checktargetiso
===================================== test session starts =====================================
platform linux -- Python 3.6.15, pytest-4.6.11, py-1.11.0, pluggy-0.13.1 -- /home/matej/projects/leapp-repo/tut/bin/python3.9
```

Is it possible that the lookup script is failing on 2.7 and 3.6?

Reply:
You are absolutely right about the definition of `_PYTHON_VENV`. Moreover, your observations about python versions led me to realize that there are some breaking changes in the `ast` since python3.8. Nevertheless, I have modified the script to work with all python versions:
```
(tut) [root@13b46a877de5 leapp-repository]# for py in python2.7 python3.6 python3.11; do ${py} utils/find_actors.py checkrhui -C repos/ ; done
Failed to parse /work/leapp-repository/repos/system_upgrade/el8toel9/actors/dotnet/actor.py. The actor might contain syntax errors, or perhaps it is written with Python3-specific syntax?
/work/leapp-repository/repos/system_upgrade/common/actors/cloud/checkrhui
/work/leapp-repository/repos/system_upgrade/common/actors/cloud/checkrhui
/work/leapp-repository/repos/system_upgrade/common/actors/cloud/checkrhui
```
The dotnet actor is using f-strings, which are python3 specific, and therefore, the is no surprise that the ast module fails to parse it. However, the original `test_no_lint` would fail as well if used with python2.7. I believe that the target should work now, can you please try @matejmatuska @fernflower?

Reply:
I can confirm that it works correctly on python 2.7, 3.6, 3.9 now.

Reply:
Yep, works like a charm!